include/moveit_visual_tools/moveit_visual_tools.h

```diff
@@ -481,13 +482,6 @@ class MoveItVisualTools : public rviz_visual_tools::RvizVisualTools
    * \param color to display the collision object with
    * \return true on sucess
    */
-  RVIZ_VISUAL_TOOLS_DEPRECATED
-  bool publishCollisionTable(double x, double y, double angle, double width, double height, double depth,
-                             const std::string name, const rviz_visual_tools::colors& color = rviz_visual_tools::GREEN)
```

Comment:
clang didn't like the name parameter, so i just removed the function as this has been deprecated for a while